tests/sanity/tests/recruiting/recruit.interview.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test } from '@playwright/test'
+import { generateId, PlatformSetting, PlatformURI } from '../utils'
+
+test.use({
+  storageState: PlatformSetting
+})
+
+test.describe('interview tests', () => {
+  test.beforeEach(async ({ page }) => {
+    // Create user and workspace
+    await (await page.goto(`${PlatformURI}/workbench/sanity-ws/recruit`))?.finished()
+  })
+
+  test('create-interview', async ({ page }) => {
```

Comment:
Please skip adding tests for reviews for now, since it will be mostly re-worked/removed and replaced by todos/calendars.

Reply:
done